### PR TITLE
Monday review

### PR DESCRIPTION
**Pull request by marwinburesch**
# Description
Team, this is a job well done! I would've just added an image of that graph, but you went the extra mile and made it happen with CSS, awesome. I can see your effort in working on a structure for the project. Good job! 🐜 🎉 

I am proposing changes, that should fix some layout and structural issues I saw, as well as some adjustment to get a proper production build going. 

Keep in mind, if you merge this some things might break 😬 but these can be fixed with ease ;)

# SUCCESS!
I can see a lot of the concepts we talked about last week, being implemented here in your app!

- I can see that you thought about CSS structure and naming conventions, nice!
- You used `@import` to import CSS in a different file than `style.css` 🥳 
- You made your own utility classes 🌈 

# General notes
## HTML
- There could be some more semantic elements in place of some `div`'s, check out [this list](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantic_elements) for some inspiration and guidance
 
## CSS
- Your naming convention for classes looks good, but is only implemented partially, in `utility.css` classes prefixed with `journal` use BEM, most other classes use another system. Some seem a bit rough :D 
- Same for splitting up your CSS, the thought was perfect, but somehow you forgot about it halfway through. Some pages have a separate stylesheet, some use only `style.css`
- The grid was only a few declarations away from not having two scroll bars :)

## Structure
- I saw some duplicate code in your CSS, some classes are declared 4 times

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,8 +3,8 @@
   "version": "0.0.0",
   "scripts": {
     "dev": "vite serve src",
-    "build": "vite build",
-    "serve": "vite preview"
+    "build": "vite build src",
+    "serve": "vite preview src"
```
Comment (marwinburesch) [inline package.json:7]: This is to create a production build from the correct entry point, the src directory

File: src/index.html [hunk 2]
```diff
@@ -104,11 +98,21 @@ <h3 class="journal__title">Yesterdays Journal</h3>
...
+          /></a>
+          <a href="teams.html"
+            ><img src="./icons/TeamIcon_empty.svg" alt=""
+          /></a>
+          <a href="graphics.html"
+            ><img src="./icons/EnergyIcon_empty.svg" alt=""
+          /></a>
+          <a href="journal.html"
+            ><img src="./icons/JournalIcon_empty.svg" alt=""
+          /></a>
```
Comment (marwinburesch) [inline src/index.html:115]: There were some extra `div`s

File: src/vite.config.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        main: resolve(__dirname, 'index.html'),
+        buddies: resolve(__dirname, 'code-buddys.html'),
+        teams: resolve(__dirname, 'teams.html'),
+        energy: resolve(__dirname, 'graphics.html'),
+        journal: resolve(__dirname, 'journal.html'),
+        journalForm: resolve(__dirname, 'form.html'),
+      },
+    },
+  },
+}
```
Comment (marwinburesch) [inline src/vite.config.js:16]: This lets the build script know to include all sub-pages to the production build

**Review by eduardoccmelo — APPROVED** [review]
Thanks for your review @marwinburesch

**Pull request merged**